### PR TITLE
fix AAP-2080

### PR DESCRIPTION
**Pull request by ferferrarazzo**

### CODE REVIEW

File: modules/ROOT/pages/install-validate.adoc [hunk 1]
```diff
@@ -6,31 +6,26 @@ endif::[]
...
-.. Open *Setup*.
-.. Search for *Named Credentials*.
-.. Locate *anypoint.mulesoft* and click *Edit*.
-.. Verify that the list *External Data User Authentications* shows your administrative Anypoint user.
-. If you are using impersonation and not JSON Web Tokens, verify the Exchange External Data Source status is *Authenticated*.
-.. Open *Setup*.
-.. Search for *External Data Sources*.
-.. In the list select *Exchange*.
-.. Verify that the *Authentication Status* is *Authenticated*.
+. Open the Salesforce Application Launcher.
```
Comment (kbostrom) [inline modules/ROOT/pages/install-validate.adoc]: It looks like this is really presented as "App Launcher," although the UI shows *App launcher*...of course....

https://www.lightningdesignsystem.com/components/app-launcher/

File: modules/ROOT/pages/install-validate.adoc [hunk 2]
```diff
@@ -6,31 +6,26 @@ endif::[]
...
+. Use the search bar to find and open the *API Community Manager Diagnostics*.
```
Comment (kbostrom) [inline modules/ROOT/pages/install-validate.adoc]: Do not precede proper nouns with "the" unless followed by a common noun. GLOBAL.
```suggestion
. Use the search bar to find and open *API Community Manager Diagnostics*.
```

File: modules/ROOT/pages/install-validate.adoc [hunk 3]
```diff
@@ -6,31 +6,26 @@ endif::[]
...
+. Check the information about your Salesforce organization in the *Organization* section.
+.. Confirm that *Package Version* shows the version of your currently installed API Community Manager package.
+.. Confirm that *Connectivity with Anypoint* shows the URL your API community uses to access Anypoint and a green check.
```
Comment (kbostrom) [inline modules/ROOT/pages/install-validate.adoc]: Never abbreviate Anypoint Platform unless quoting syntax:
```suggestion
.. Confirm that *Connectivity with Anypoint* shows the URL your API community uses to access Anypoint Platform and a green check.
```

Comment (kbostrom) [reply modules/ROOT/pages/install-validate.adoc]: Note that at some point we want to start removing reference to color, for accessibility. GLOBAL consideration.

File: modules/ROOT/pages/install-validate.adoc [hunk 4]
```diff
@@ -6,31 +6,26 @@ endif::[]
...
 +
-image::validate-exchange-auth.png[Exchange Authenticated]
+If *Connectivity with Anypoint* does not show the URL and a green check, perform the xref:installation-prerequisites.adoc[prerequisite steps] and then repeat the installation.
+.. Confirm that *Cache* shows that the cache is configured.
 +
```
Comment (kbostrom) [inline modules/ROOT/pages/install-validate.adoc:17]: Will it be clear to the audience what to look for to verify configuration (or not)?

File: modules/ROOT/pages/install-validate.adoc [hunk 5]
```diff
@@ -6,31 +6,26 @@ endif::[]
...
-. Verify that API versions are shown in the API Community Manager control panel. 
-.. Open the Application Launcher.
-.. Enter *API Community Manager*.
-.. Verify that the list *APIs in Exchange* is not empty and contains the APIs in your organization.
+If *Cache* shows that the cache is not configured, xref:cache.adoc[configure the platform cache].
+. Check the information about each of your communities in the *ACM Communities* section.
 +
-image::apis-in-exchange.png[APIs in Exchange]
+Each community shows *Community status*, whether the community is *Public*, and the *Published assets status*, and options to open the community or open the community builder. If some of your communities are not public, or some of your communities are not yet implemented, then this information is visible here.
```
Comment (kbostrom) [inline modules/ROOT/pages/install-validate.adoc]:
```suggestion
Each community shows *Community status*, whether the community is *Public*, the *Published assets status*, and options to open the community or open the community builder. If some of your communities are not public, or some of your communities are not yet implemented, then this information is visible here.
```

**Review by kbostrom — DISMISSED** [review]
CE changed text

File: modules/ROOT/pages/install-validate.adoc (hunk 1 above)
Comment (wwarner-salesforce) [reply modules/ROOT/pages/install-validate.adoc]: Good point! Updated to "App".

**Review by kbostrom — APPROVED** [review]
RA

**Pull request merged**